packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -16,6 +16,9 @@ import { UriSelection } from '@theia/filesystem/lib/common/filesystem-selection'
 import { SingleTextInputDialog, ConfirmDialog } from "@theia/core/lib/browser/dialogs";
 import { OpenerService, OpenHandler, open } from "@theia/core/lib/browser";
 import { WorkspaceServer } from '../common/workspace-protocol';
+import { localize as _localize } from './localize';
+const localize = _localize.bind(undefined, "workspace-commands");
```

Comment:
Could we actually wrap this up in something easier to understand like _localise.withPrefix("workspace-command") and thus hide the bind thing...

Reply:
Done, created a new function : localizeWithPrefix to encapsulate the bind function with localize function